server/apps/postgres-app/sample-configuration-single/search.properties

```diff
@@ -0,0 +1 @@
+implementation=scanning
```

Comment:
Add a comment: `not for production purposes. To be replaced by PG based search.`